challenges/pt-br/back-challenge.md

```diff
@@ -48,20 +52,31 @@ Usuário exemplo:
 }
 ```
 
-* As regras de descontos da aplicação são:
-  * Se for aniversário do usuário, o produto terá 5% de desconto
-  * Se for black friday (nesse exemplo ela pode ser fixada dia 25/11) o produto terá 10% de desconto
-  * O desconto não pode passar de 10%
+The returned discount **must** obey the following rules:
+  * If it's the user’s birthday, the product has 5% discount.
+  * If it is black friday (for this test you can assume BlackFriday is November 25th), the product has 10% discount
+  * No product discount can be bigger than 10%
 
+  
+  ### Service 2: Products list
 
-## Serviço 2: Listagem de produtos
-* Expõe uma rota HTTP tal que `GET /product` retorne um json com uma
-lista de produtos.
+This service exposes a HTTP endpoint `GET /product` that returns a list of products.
 
-* Essa rota deve receber opcionalmente via header `X-USER-ID` um id de usuário.
+1. The endpoint response **must** be `JSON` encoded.
+2. To calculate each product discount the service **must** consume service 1 via gRPC.
+3. If service 1 errors while calculating a discount, the service **must** returns the product list but with zero discounts for the affected products.
 
-* Para obter o desconto personalizado este serviço deve utilizar o serviço 1.
+## Evaluation Criteria
+​
+1. The problems are solved efficiently and effectively, the application works as expected.
+2. The application is supplied with the setup scripts. Consider using docker and a one-liner setup.
+3. You demonstrate the knowledge on how to test the critical parts of the application. We **do not require** 100% coverage.
+4. The application is well and logically organised.
+5. The submission is accompanied by documentation with the reasoning on the decisions taken.
+6. The code is documented and is easy to follow.
+7. Following the industry standard style guide.
+8. A git history (even if brief) with clear, concise commit messages.
+​
 
-* Caso o serviço 1 retorne um erro, a lista de produtos ainda precisa ser retornada, porém com esse produto que deu erro sem desconto.
+Before you start, take a look at our [key values](https://www.keyvalues.com/hash). It can show you how we think and work in terms of development. Also do your best! We'll evaluate it like you did.
```

Comment:
```suggestion
Also if you'd like to more about our developer culture, how we think and work in terms of development. check out our  [key values](https://www.keyvalues.com/hash).
```

Reply:
> 

Also if you'd like to _know_ more about our developer culture, how we think and work in terms of development. check out our  [key values](https://www.keyvalues.com/hash).

Reply:
Fixed!